metaflow/plugins/cards/card_datastore.py

```diff
@@ -74,6 +79,20 @@ def __init__(self, flow_datastore, pathspec=None):
         self._run_id = run_id
         self._step_name = step_name
         self._pathspec = pathspec
+
+        # Save the task's metaflow version so that we can use it in the `_make_path` function.
+        # Since metaflow version >= 2.7.8 had a change in the directory structure for cards, CardDatastore object needs this information to determine the correct directory structure for retrieving/storing cards.
+        from metaflow.client import Run
```

Comment:
Can we avoid making the call to the metadata service from within this module? Is this call strictly needed?

Reply:
I spoke to Romain and I will change this. I will be doing a small refactor to remove the `Run` object altogether.